### ISSUE TITLE
Add Share button to Post Page

### ISSUE THREAD

**Issue by derecklhw** [cloud-native-mauritius/cloudnativemauritius.com#3]

**Comment by eznix86** [comment]
We can use: 
- https://github.com/cybercog/laravel-love
- https://github.com/jorenvh/laravel-share

For comments, can be implemented in different ways, and other packages out there are not solid

**Comment by ish-sookun** [comment]
Social media sharing button are fine.
Not much in favour of comments though. It's additional work to "review/moderate", prevent spam and some times it can cause never ending discussions. I'd prefer such discussions to be on social media, e.g X or LinkedIn. What do you folks think?

**Comment by eznix86** [comment]
Yes, i prefer comments not to be implemented too and for `Add Like` is it for anybody to like or you need to have an account to like ?

**Comment by derecklhw** [comment]
I think it might be better for someone to have an account, else there will be a lot of spamming of likes.

**Comment by eznix86** [comment]
I think better wait for #2 to finish first

**Comment by ish-sookun** [comment]
I'd prefer not having people to create an account to submit for the CfP. They simply fill the form with firstname, lastname, email, title, description, job title (optional), 'company (optional)', etc. The form should use the [Cloudflare Captcha](https://www.cloudflare.com/en-gb/products/turnstile/).

I would discourage having the "like" feature for the blog post too. Let's just have simple social media sharing. People should share and express on social media instead of putting likes, "dislikes" or comments directly on the site.

**Comment by eznix86** [comment]
Thanks @ishwon so i think this issue can be closed and #2 too

**Comment by derecklhw** [comment]
I will just rename it for reshare them ? @ishwon @eznix86 

#2 was for the user management of the admin page, like forget password XD

**Comment by eznix86** [comment]
i believe we could do it by using laravel jetstream for user management.

<img width="272" alt="Screenshot 2024-08-27 at 18 45 48" src="https://github.com/user-attachments/assets/ffcdde08-f091-461e-841b-27c6fa4db925">

We will get: `App\Actions\Jetstream\AddTeamMember`

**Comment by ish-sookun** [comment]
We're using FilamentPHP. We just need to create the `UserResource` and add the appropriate columns.

```
php artisan make:filament-resource User --view
```

You can then head to the `UserResource.php` file and add the columns to manage users.
